README.md

```diff
@@ -58,19 +58,48 @@ Provision a Kubernetes cluster. Again, if you are using the IKS, you cannot use
 - can not leverage a persistent volume
 - can not leverage an ingress controller
 
-In Terminal, log into IBM Cloud with `ibmcloud login`. Run `ibmcloud ks clusters` to get the name of your cluster:
+In Terminal, log into IBM Cloud with `ibmcloud login`.
 
-```bash
-testing-cluster           bkss2low0aucnromsrm0               normal   2 weeks ago   1         Washington D.C.   1.13.8_1529   dokun-personal
+```console
+$ ibmcloud login -u <username> -p <password>
+$ ibmcloud target --cf
+$ ibmcloud target -g Default
+```
+
+Create a Classic Cluster,
+```console
+$ ibmcloud ks zones
+$ ibmcloud ks machine-types --zone dal10
+$ ibmcloud ks cluster create classic --name <account_name>_iks_cluster_<number> --zone dal10 --machine-type b3c.4x16 --hardware shared --workers 3 
```

Comment:
I sincerely wonder if creating a cluster via the command line is out of scope for *this* tutorial. However, I like that it’s written out, which means its scriptable to the liking of the person running the workshop, so I would suggest perhaps making another readme with a sample script for creating clusters?